doc/api/http.md

```diff
@@ -841,8 +841,25 @@ Note that the socket timeout logic is set up on connection, so
 changing this value only affects *new* connections to the server, not
 any existing connections.
 
-Set to 0 to disable any kind of automatic timeout behavior on incoming
-connections.
+Set to 0 to disable timeout behavior on incoming connections.
```

Comment:
[Suggestion] /s/Set to 0 to disable/A value of 0 will disable

P.S. what does it mean? Sockets will be destroyed ASAP, or timeout is delegated to OS?